README.md

```diff
@@ -142,44 +142,60 @@ docker container exec -i 12cf98736487 bash
+ * [Switch AWS Web Console](https://ap-northeast-2.console.aws.amazon.com/ec2/v2/home?region=ap-northeast-2#Instances:sort=instanceId) to region to which you want to set up new instance (top right corner, next to Support).
  * Create network infrastructure:
     * Define [VPC with Elastic IP](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/create-public-private-vpc.html).
     * Define *security group* using previously created *VPC*.
     * Add 22 and 3000 ports to *Inboud Roules* of newly created security group.
  * [Create Your Cluster](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-cli-tutorial-ec2.html), for example, 
 ```bash
-ecs-cli up --force --keypair id_rsa --capability-iam --size 1 --instance-type t2.large --vpc vpc-0d05d256d9261ccb5 --subnets subnet-0b31dfed2f9dddb0a --security-group sg-09d2b747ca8b77f1a --cluster-config watt-cluster-config --region ap-northeast-2
+ecs-cli up --force --keypair id_rsa --capability-iam --size 1 --instance-type t2.large --vpc vpc-0d05d256d9261ccb5 --subnets subnet-0b31dfed2f9dddb0a --security-group sg-09d2b747ca8b77f1a --cluster-config watt-cluster-config --region REGION_CODE
 ```
- * To find subnet id go to [Subnet dashboard](https://ap-northeast-2.console.aws.amazon.com/vpc/home?region=ap-northeast-2#subnets:sort=SubnetId) and copy public subnet id associated with newly created VPC.
-
-## Deploying the compose file to a cluster
- * You can omit steeps below in order to use already created task definition based on compose file.
- * Update docker-compose-aws.yml with new docker images repositories.
+ * To find subnet id go to [Subnet dashboard](https://ap-northeast-2.console.aws.amazon.com/vpc/home?region=ap-northeast-2#subnets:sort=SubnetId) change the region and copy public subnet id associated with newly created VPC.
+ * Get WATT and mongod images using the following script:
+```bash
+./docker-run.sh --rebuild
+```
+ * You don't have to necessarily wait for complete WATT set up in docker environment. It's enough to have required images that can be checked by:
+```bash
+docker images
+```
+ * Visit [docker repositories](https://ap-northeast-2.console.aws.amazon.com/ecr/repositories?region=ap-northeast-2#), change the region and create repositories for WATT and mongod images.
+ * Update docker-compose-aws.yml with new docker images repositories and awslogs-region.
  * You can change default memory limits for each container in ecs-params.yml
  * [Deploy the Compose File to a Cluster](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-cli-tutorial-ec2.html#ECS_CLI_tutorial_compose_deploy), for example,
 ```bash
-ecs-cli compose --file docker-compose-aws.yml --verbose up --create-log-groups --cluster-config watt-cluster-config --region ap-northeast-2
+ecs-cli compose --file docker-compose-aws.yml --verbose up --create-log-groups --cluster-config watt-cluster-config --region REGION_CODE
```

Comment:
What about adding short info what regions now we are supporting?

Reply:
Added separate section with list of our currently maintaining instances.